MdeModulePkg/Library/UefiBootManagerLib/BmLoadOption.c

```diff
@@ -166,6 +166,12 @@ BmGetFreeOptionNumber (
   Create the Boot####, Driver####, SysPrep####, PlatformRecovery#### variable
   from the load option.
 
+  // MU_CHANGE START
+  If the OptionNumber is LoadOptionTypePlatformRecovery and the call to
+  SetVariable was successful, a variable policy will be registered for
+  the PlatformRecovery#### variable.

```

Comment:
```suggestion
  If the OptionNumber is LoadOptionTypePlatformRecovery and the call to
  SetVariable was successful, a lock now variable policy will be registered
  for the PlatformRecovery#### variable.
```

Reply:
This won't be visible in the header file. That might be okay since this is an instance implementation-specific detail.